docs/functions/functions.json

```diff
@@ -538,6 +626,45 @@
 				true
 			]
 		},
+		{
+			"name": "intersects",
+			"id": "intersects",
+			"unary": false,
+			"category": "lists",
+			"role": "function",
+			"description": "Checks if two lists share one or more of the same item.",
+			"argument": [
+				{
+					"name": "list1",
+					"optional": false,
+					"type": "List"
+				},
+				{
+					"name": "list2",
+					"optional": false,
+					"type": "List"
+				}
+			],
+			"returnDescription": "True if there is an intersection and false if not.",
+			"demo": "`intersects([1,2,3], [4,3,5])` will be true because both share the item `3`."
```

Comment:
This is not what the `"demo"` parameter is for. You can list examples in the description. The `demo` parameter is for an actual live demo.
Please check another function with a demo for an example of how to use this.